tests/integration/update.bats

```diff
@@ -354,6 +354,36 @@ EOF
     check_systemd_value "CPUShares" 100
 }
 
+@test "update cgroup v2 cpu limits" {
```

Comment:
I don't think it makes sense to have separate tests for v1 and v2 -- runc should work the same way in both, only the check it different but I was able to unify the checks in https://github.com/opencontainers/runc/pull/2428/files#diff-4c21eb8cb6048713e619964ad6d1cab9.